src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+
+{{site.data.alerts.callout_info}}
+If you would like to ignore TTL deletes in the LDR stream, you can use the `discard = ttl-deletes` option in the `CREATE LOGICAL REPLICATION STREAM` statement.
+{{site.data.alerts.end}}
+
+1. From the destination cluster, start the LDR stream. Use the fully qualified table name and ensure the source and destination table names are identical:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+    You can change the default `mode` using the `WITH mode = validated` syntax.
+
+    Once the LDR stream has started, an LDR job will start on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use `SHOW LOGICAL REPLICATION JOBS` to display the LDR job IDs:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW LOGICAL REPLICATION JOBS;
+    ~~~
+    ~~~
+            job_id        | status  |          targets          | replicated_time
+    ----------------------+---------+---------------------------+------------------
+    1012877040439033857   | running | {database.public.table}   | NULL
+    (1 row)
+    ~~~
+
+    If you're setting up bidirectional LDR streams, both clusters will have a history retention job and an LDR job running.
+
+1. Move on to [Step 4](#step-4-optional-set-up-bidirectional-ldr-streams) to set up a second LDR stream. Or, once you have set up your required LDR streams, refer to [Step 5](#step-5-monitor-the-ldr-jobs) to monitor the jobs in the DB Console.
+
+## Step 4. (Optional) Set up bidirectional LDR streams
+
+At this point, you've set up one LDR stream from cluster A as the source to cluster B as the destination. 
+
+1. To set up a stream flowing in the opposite direction, complete [Step 2](#step-2-connect-from-the-destination-to-the-source) and [Step 3](#step-3-start-the-ldr-stream) again. Cluster B will now be the source, and cluster A will be the destination.
+1. After the two streams are running, you can check the number of rows on each cluster with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT COUNT(*) from {table_name};
```

Comment:
did you add this primarily so the user can see if rows are replicating?

Reply:
OK, for now, took that out. Will build out this step with the verification tooling etc, when I'm working on the PR for monitoring LDR.